.env-template

```diff
@@ -1,2 +1,12 @@
 MYRIAD_WEBSITE_URL=
-MYRIAD_RPC_URL=
\ No newline at end of file
+MYRIAD_RPC_URL=
```

Comment:
change this to `NEXT_PUBLIC_MYRIAD_RPC_URL` and put this above `NEXT_PUBLIC_MYRIAD_API_URL`